tests/dags/common/sensors/test_single_run_external_dags_sensor.py

```diff
@@ -17,13 +17,20 @@
 TEST_TASK_ID = "wait_task"
 TEST_POOL = "single_run_external_dags_sensor_test_pool"
 DEV_NULL = "/dev/null"
+DAG_PREFIX = "sreds"  # single_run_external_dags_sensor
 
 
 @pytest.fixture(autouse=True)
 def clean_db():
     with create_session() as session:
-        session.query(DagRun).delete()
-        session.query(TaskInstance).delete()
+        # synchronize_session='fetch' required here to refresh models
+        # https://stackoverflow.com/a/51222378 CC BY-SA 4.0
```

Comment:
Noticing now that SO uses a CC license! I'll keep that in mind for when I copy links too 💯

Reply:
I saw someone else do it (I think maybe @sarayourfriend?) and I loved the idea 😄